src/common_subset.rs

```diff
@@ -1,4 +1,26 @@
-//! Asynchronous Common Subset algorithm.
+//! # Asynchronous Common Subset algorithm.
+//!
+//! The Asynchronous Common Subset protocol assumes a network of _N_ nodes that send signed
+//! messages to each other, with at most _f_ of them malicious, where _3 f < N_. Handling the
+//! networking and signing is the responsibility of the user: only when a message has been
+//! verified to be "from node i", it can be handed to the `CommonSubset` instance.
```

Comment:
We discussed dropping Common and renaming to simply Subset. If too vague, can keep as CommonSubset and update readme. Same for Common Coin renamed to Coin.

Reply:
Yes, and we also need to rename all the identifiers in the code (https://github.com/poanetwork/hbbft/issues/72). But maybe it's better to postpone this a few more days, otherwise @vkomenda's API redesign will have an even bigger merge conflict.
Let's do the renaming after that, in a separate PR, for all the code and docs at once.